hledger-lib/Hledger/Data/Transaction.hs

```diff
@@ -132,7 +132,7 @@ Render a journal transaction as text in the style of Ledger's print command.
 Ledger 2.x's standard format looks like this:
 
 @
-yyyy/mm/dd[ *][ CODE] description.........          [  ; comment...............]
+yyyy-mm-dd[ *][ CODE] description.........          [  ; comment...............]
```

Comment:
Not correct I think, both Ledger 2 and 3 print dates with slashes.

Reply:
Oh, right. But this is the documentation for the showTransaction function, so we also have to fix the sentence above my change.

Reply:
I changed the above part of comment.